src/aks/subscriptions/PROD-CSTAR/terraform.tfvars

```diff
@@ -7,60 +7,78 @@ ingress_load_balancer_ip = "x.x.x.x"
+  BPD_CITIZEN_CHECKIBAN_PROXY_HOST                  = "x.x.x.x" # TODO
+  BPD_CITIZEN_CHECKIBAN_PROXY_PORT                  = "8181"
+  BPD_CITIZEN_SERVER_THREAD_MAX                     = "350"
+  CITIZEN_SECONDARY_DB_ENABLE                       = "true"
+  KAFKA_CITIZENTRX_TOPIC                            = ""
+  KAFKA_POINTTRX_TOPIC                              = ""
+  LOG_LEVEL_CITIZEN                                 = "INFO"
+  PAGOPA_CHECKIBAN_URL                              = "https://bankingservices.pagopa.it"
+  POSTGRES_POOLSIZE                                 = "20"
+  POSTGRES_REPLICA_POOLSIZE                         = "10"
+  POSTGRES_SHOW_SQL                                 = "false"
+  POSTGRES_TIMEOUT                                  = "10000"
 }
 
 # bpdmscitizenbatch
 configmaps_bpdmscitizenbatch = {
-  BPD_CITIZEN_CHECKIBAN_PROXY_ENABLE = "false"
-  CITIZEN_SECONDARY_DB_ENABLE        = "true"
-  KAFKA_CITIZENTRX_ENABLE            = "true"
-  KAFKA_POINTTRX_ENABLE              = "true"
-  KAFKA_SESSION_TIMEOUT              = "10000"
-  LOG_LEVEL_CITIZEN                  = "INFO"
-  POSTGRES_REPLICA_POOLSIZE          = "20"
-  POSTGRES_SHOW_SQL                  = "false"
+  JAVA_TOOL_OPTIONS                                 = "-javaagent:/applicationinsights-agent-3.1.0.jar"
+  APPLICATIONINSIGHTS_ROLE_NAME                     = "bpdmscitizenbatch"
+  APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL = "OFF"
+  BPD_CITIZEN_CHECKIBAN_PROXY_ENABLE                = "false"
+  CITIZEN_SECONDARY_DB_ENABLE                       = "true"
+  KAFKA_CITIZENTRX_ENABLE                           = "true"
+  KAFKA_POINTTRX_ENABLE                             = "true"
+  KAFKA_SESSION_TIMEOUT                             = "10000"
+  LOG_LEVEL_CITIZEN                                 = "INFO"
+  POSTGRES_REPLICA_POOLSIZE                         = "20"
+  POSTGRES_SHOW_SQL                                 = "false"
 }
 
 # bpdmsenrollment
 configmaps_bpdmsenrollment = {
-  LOG_LEVEL_FA_ENROLLMENT          = "INFO"
-  BPD_ENROLLMENT_SERVER_THREAD_MAX = "400"
+  JAVA_TOOL_OPTIONS                                 = "-javaagent:/applicationinsights-agent-3.1.0.jar"
+  APPLICATIONINSIGHTS_ROLE_NAME                     = "bpdmsenrollment"
+  APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL = "OFF"
+  LOG_LEVEL_FA_ENROLLMENT                           = "INFO"
+  BPD_ENROLLMENT_SERVER_THREAD_MAX                  = "400"
 }
 
 # bpdmsnotificationmanager
 configmaps_bpdmsnotificationmanager = {
+  JAVA_TOOL_OPTIONS                                                = "-javaagent:/applicationinsights-agent-3.1.0.jar"
+  APPLICATIONINSIGHTS_ROLE_NAME                                    = "bpdmsnotificationmanager"
+  APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL                = "OFF"
   BPD_AWARD_PERIOD_HOST                                            = "bpdmsawardperiod"
   JAVA_TOOL_OPTIONS                                                = "-Xms256m -Xmx6g"
```

Comment:
Here you need to have only one JAVA_TOOL_OPTIONS "-Xms256m -Xmx6g -javaagent:/applicationinsights-agent-3.1.0.jar"

Reply:
ops copy and paste <3